app/pdl.yaml

```diff
@@ -447,4 +467,4 @@ components:
           type: number
         all_programs_total_18_22:
           nullable: true
-          type: number
+          type: number
```

Comment:
Ending the file with a blank new line is good practice. Please see more details here. https://thoughtbot.com/blog/no-newline-at-end-of-file. The rest looks fine to me.

Reply:
added empty line

Reply:
Thanks.